### DISCUSSION TITLE
Development changes

### DISCUSSION THREAD

**Discussion by MarkoSipovac** [MarkoSipovac/Tafe22S216#5] [General]
Hey just letting you guys know I've added my section of the program both to a new branch and now have merged it under the Development branch

**Comment by Kristian-HD** [comment]
Cool will look into.

**Comment by Kristian-HD** [comment]
so when it says to "merge from the feature branch into the development branch" should I merge with the "Development" or the "development" branch

**Reply by MarkoSipovac** [reply]
Mine is in Development so id say that one so that all the features are together in the one file. that sounds about right yeah?

**Reply by Kristian-HD** [reply]
i did the merge with the "Development" branch

**Comment by MarkoSipovac** [comment]
Arthur any chance you are able to put your part of the assignment into this (section C) as that is holding us up and this was due last week
